test/e2e-ocl/onclusterlayering_test.go

```diff
@@ -572,7 +577,8 @@ func TestMCDGetsMachineOSConfigSecrets(t *testing.T) {
 	})
 
 	// Assign the secret name to the MachineOSConfig.
-	mosc.Spec.BuildOutputs.CurrentImagePullSecret.Name = secretName
+	// QOCL: This field doesn't exist anymore, is this test valid?
+	//mosc.Spec.CurrentImagePullSecret.Name = secretName
```

Comment:
Another scenario where I believe the test currently depends on this field to run correctly

Reply:
This whole test can be dropped in favor of a test that just validates whether the global image pull secret is written to the nodes' filesystem.

Reply:
I believe we already do this as part of the internal registry secret cert writer path, where the global pull secret +  internal registry secret is written to disk at `/etc/mco/internal-registry-pull-secret.json` : https://github.com/openshift/machine-config-operator/pull/4395/commits/2d77dcf4a9b8817f8e58026e31346056f22570f8